src/modules/LanItems.rb

```diff
@@ -276,6 +276,14 @@ def GetDeviceNames(items)
       items.map { |itemId| GetDeviceName(itemId) }.reject(&:empty?)
     end
 
+    def current_name
+      renamed?(@current) ? renamed_to(@current) : GetDeviceName(@current)
+    end
+
+    def current_device_names
+      GetNetcardInterfaces().map { |i| renamed?(i) ? renamed_to(i) : GetDeviceName(i) }.reject(&:empty?)
```

Comment:
NP: A bit difficult to read a first glance (for me)

Even more, you are using the same logic twice

```ruby
 renamed?(i) ? renamed_to(i) : GetDeviceName(i)
```

Why not 

```ruby

...

private

...

def name_for(device)
  renamed?(device) ? renamed_to(device) : GetDeviceName(device)
end
```

and then

```ruby
...

def current_name
  name_for(@current)
end

...

def current_device_names
  GetNetcardInterfaces().map { |device| name_for(device) }.reject(&:empty?)
end

...
```